src/scss/components/navigation/_index.scss

```diff
@@ -93,3 +99,32 @@
     }
   }//hx-reveal
 }
+
+// ===== SUB-LEFT NAVIGATION =====
```

Comment:
This is better and very close!

In the screenshot, please see:
* `Primary Selected` is bolder font when selected.
* `Secondary Selected`  has normal font when selected.
*  Please verify we have all nav states (e.g, disabled).
*  Please verify the proper padding when `selected`.
*  The specs include an example `w/ Icon`.  We should have styles for these.

<img width="905" alt="Screen Shot 2021-04-28 at 11 42 16 PM" src="https://user-images.githubusercontent.com/10120600/116504340-a9517b80-a87d-11eb-96af-d2233b13cb17.png">